sidecars/rust/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rust:1.48.0-slim-buster
+
+ENV HOME=/home/theia
+
+RUN mkdir /projects ${HOME} && \
+    # Change permissions to let any arbitrary user
+    for f in "${HOME}" "/etc/passwd" "/projects"; do \
+      echo "Changing permissions on ${f}" && chgrp -R 0 ${f} && \
+      chmod -R g+rwX ${f}; \
+    done && \
+    rustup component add rust-src; \
```

Comment:
out of curiosity: how did we ended it up with these 3 components ?
what if they're not added at all

Reply:
These are required by the rust extension.
The rust extension is trying to install them at startup through provided dynamic task but fails in Che-theia (failed to run provided tasks, i will create an issue afterward, as it cannot run other provided tasks neither)

Reply:
these are required by the extension: https://github.com/rust-lang/vscode-rust/blob/master/src/rls.ts#L17
the extension is trying to install them if not present. but fails to run the tasks dynamically in Che-theia.
I will create an issue about that afterwards as some other provided tasks are failing to run.